### DISCUSSION TITLE
Headerfilter on one column

### DISCUSSION THREAD

**Discussion by paukyduckyKore** [komarovalexander/ka-table#188] [Q&A] [answered]
Is there a way to use Headerfilter on only one column instead of all?

**Comment by komarovalexander** [comment, marked as answer]
At this time this is possible only by css. You can add custom class to headCell where you want it to be hidden. Thanks for the question, next year I will add some built in setting to do that

**Reply by paukyduckyKore** [reply]
Thank you!
